build.sh

```diff
@@ -4,12 +4,14 @@
 usage() {
     echo "Usage: $0 [-l] [-f <ftl_branch>] [-c <core_branch>] [-w <web_branch>] [-t <tag>] [use_cache]"
     echo "Options:"
-    echo "  -f, --ftlbranch <branch>     Specify FTL branch (cannot be used in conjunction with -l)"
-    echo "  -c, --corebranch <branch>    Specify Core branch"
-    echo "  -w, --webbranch <branch>     Specify Web branch"
-    echo "  -t, --tag <tag>              Specify Docker image tag (default: pihole)"
-    echo "  -l, --local                  Use locally built FTL binary (requires src/pihole-FTL file)"
-    echo "  use_cache                    Enable caching (by default --no-cache is used)"
+    echo "  -f, --ftlbranch <branch>       Specify FTL branch (cannot be used in conjunction with -l)"
+    echo "  -c, --corebranch <branch>      Specify Core branch"
+    echo "  -w, --webbranch <branch>       Specify Web branch"
+    echo "  -t, --tag <tag>                Specify Docker image tag (default: pihole)"
+    echo "  -l, --local                    Use locally built FTL binary (requires src/pihole-FTL file)"
+    echo "  -v, --dockerversion <version>  Specify a version number for the image (default: dev-unknown - must match following pattern: https://regex101.com/r/RsENuz/1)"
```

Comment:
I think `dockerversion` will cause confusion. This is the image version.

```suggestion
    echo "  -v, --imageversion <version>  Specify a version number for the image (default: dev-unknown - must match following pattern: https://regex101.com/r/RsENuz/1)"
```